### PR TITLE
Made Modules in List responsive

### PR DESCRIPTION
**Pull request by rheactdev**
# What's here (& why)

Responsive design added to the module list.
- https://trello.com/c/qOkJaV21/14-as-a-user-i-can-enjoy-a-responsive-design-at-both-the-desktop-and-mobile-levels-for-every-page

# Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete / work-in-progress, PR is for discussion or feedback
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How has this been tested?

- [x] Tested locally
- [x] Test suite provides the same results as prior to making changes

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/UserDashboard/UserCard.js [hunk 1]
```diff
@@ -40,8 +40,8 @@ export default function UserCard(props) {
   return (
     <div className="user-card">
       {<Dropdown.Button overlay={menu}></Dropdown.Button>}
-      {user.userid}, {user.username}, {user.firstname},{' '}
-      {user.roles[0].role.name}
+      {console.log(user)}
```
Comment (BUGBOUNTYchrisg8691) [inline src/components/UserDashboard/UserCard.js]: Could you get rid of this `console.log` before we merge this, please? Just remove locally and re-commit and push on this same branch and mark this convo as resolved, and we should be good to go.

**Review by BUGBOUNTYchrisg8691 — DISMISSED** [review]
Just the one fix and we are good.

File: src/components/UserDashboard/UserCard.js (hunk 1 above)
Comment (chazkiker2) [reply src/components/UserDashboard/UserCard.js]: Agreed on this front, everything looks good otherwise!

**Review by chazkiker2 — APPROVED** [review]
Completely agree with Chris! But looks good!

**Review by chazkiker2 — APPROVED** [review]
Looks great, I think this should be ready to emrge

**Review by chazkiker2 — APPROVED** [review]
reviewing again b/c GitHub thought my newest approve was stale for some reason.

**Review by sebass482 — APPROVED** [review]

**Review by simonesquad — APPROVED** [review]
Media Breakpoint approved

**Review by simonesquad — APPROVED** [review]
conversation resolved

**Pull request merged**